File: 백준/Bronze/3003. 킹， 퀸， 룩， 비숍， 나이트， 폰/킹， 퀸， 룩， 비숍， 나이트， 폰.py
import sys 
input = sys.stdin.readline
sys.setrecursionlimit(10**6)
right = [1,1,2,2,2,8]
search = list(map(int, input().split()))
for i, v in enumerate(right):
    right[i]=v-search[i]
    print(right[i], end=' ')

File: 백준/Bronze/3003. 킹， 퀸， 룩， 비숍， 나이트， 폰/킹， 퀸， 룩， 비숍， 나이트， 폰.go
package main

import (
	"bufio"
	"fmt"
	"os"
)

func main() {
	reader := bufio.NewReader(os.Stdin)
	var right = []int{1, 1, 2, 2, 2, 8}
	var search = make([]int, 6)
	for i := 0; i < 6; i++ {
		fmt.Fscanf(reader, "%d ", &search[i])
	}
	for i := 0; i < 6; i++ {
		right[i] = right[i] - search[i]
		fmt.Printf("%d ", right[i])
	}
}
